collections/_version2-english/9-fork/1-basic-knowledge.md

```diff
@@ -117,3 +126,15 @@ In this case, the threshold (`275,000` REP) is exceeded in Dispute Round 20. Tha
 
 ## When does a fork end?
 The period from the start to the end of a fork is referred to as the [forking period]({{glossary_forking_period}}). The forking period ends when either 60 days have passed, or more than 50% of [all theoretical REP]({{glossary_all_theoretical_rep}}) is migrated to one of the [child universes]({{glossary_child_universe}}). Whichever child universe receives the most migrated REP by the end of the forking period becomes the [winning universe]({{glossary_winning_universe}}), the other child universes become the [losing universe]({{glossary_losing_universe}}).
+
+## What happens to the parent universe?
+When a fork starts the [parent universe]({{glossary_parent_universe}}) becomes permanently locked. In the [locked universe]({{glossary_locked_universe}}), no new [markets]({{glossary_market}}) can be created and no markets can go through the [reporting]({{glossary_report}}) process. As a result of this, no markets can be [finalized](glossary_finalized_market). (See [Restrictions on use]({{url_restrictions_on_use}}) for details.)
```

Comment:
```suggestion
When a fork starts, the [parent universe]({{glossary_parent_universe}}) becomes permanently locked. In the [locked universe]({{glossary_locked_universe}}), no new [markets]({{glossary_market}}) can be created and no markets can go through the [reporting]({{glossary_report}}) process. As a result of this, no markets can be [finalized](glossary_finalized_market). (See [restrictions on use]({{url_restrictions_on_use}}) for details.)
```

Reply:
@MicahZoltu  I capitalized the first letter because it's the title of the page.

 - To-do's
 - Restrictions on use
 - Migration of the objects
 
These are the titles of the pages that I will add to the fork section. Should these titles generally start with a lowercase letter?

Reply:
Usually inline links like that have a casing (lowercase vs uppercase) that aligns with the flow of the document they are part of.  They tend to not follow traditional Title Case rules like when referring to the title of a book or movie.

Reply:
Oh, I see 😀 There are other links that need to be modified like this. I will submit it as another pull request.